cartridges/int_yotpo_sfra/cartridge/models/common/yotpoConfigurationModel.js

```diff
@@ -134,8 +134,8 @@ function setYotpoConfigOverrides(locale) {
             if ('enablePurchaseFeed' in localeConfigObject.custom) {
                 yotpoConfig.enablePurchaseFeed = localeConfigObject.custom.enablePurchaseFeed;
             }
-            if ('productInformationFromMaster' in localeConfigObject.custom) {
-                yotpoConfig.productInformationFromMaster = localeConfigObject.custom.productInformationFromMaster;
+            if ('yotpoProductInformationFromMaster' in localeConfigObject.custom) {
+                yotpoConfig.yotpoProductInformationFromMaster = localeConfigObject.custom.yotpoProductInformationFromMaster;
```

Comment:
Interestingly, the `localeConfigObject` originates from Custom Object with `type-id=yotpoConfiguration`. That Custom Object defined in `Metadata\metadata\meta\custom-objecttype-definitions.xml` has **no** definition for the attribute `yotpoProductInformationFromMaster` (and no trace of the old attribute productInformationFromMaster). While this is safe code because of the `in` check, these lines will never be hit unless the actual object definition is updated.